### ISSUE TITLE
line macro interferes with assert macro

### ISSUE THREAD

**Issue by jgrund** [David-OConnor/seed#45]
Consider (using seed v0.2.6):

```rust
#[macro_use]
extern crate seed;

fn main () {
    assert!(true);
}
```

`cargo expand` displays:

```rust
fn main() {
    if !true {

        {
            ::std::rt::begin_panic("assertion failed: true",
                                   &("iml-action-dropdown/src/lib.rs",
                                     {
                                         let mut el =
                                             El::empty_svg(seed::dom_types::Tag::Line);
                                         el
                                     }, 5u32))
        }
    };
}
```

I've tracked this down to the `line!` macro.

If we do not import the line! macro, the assert expands as expected.

**Comment by David-OConnor** [comment]
Nice find re narrowing it down. Appears to occur with any macro named line. Can reproduce by making a new crate with `cargo new cratename --bin`, with this as `main.rs`:
```rust
#[macro_export]
macro_rules! line {
    ( a:expr ) => {

    }
}

fn main() {
    assert!(true);
}
[/code]

In next commit/release, have renamed the `line!` macro to `line_!`.

**Comment by johnsonw** [comment]
👍 Any idea why the macro gets expanded this way?

**Comment by jgrund** [comment]
@johnsonw It appears to be because the `assert!` macro is invoking the [`line!`](https://doc.rust-lang.org/std/macro.line.html) macro to get the line the assert appears on.

When the seed `line!` macro is in scope, it's overwriting the builtin one.

**Comment by johnsonw** [comment]
Understood. That makes sense.

**Comment by jordwest** [comment]
I'm seeing the same error with `dbg!()` and `panic!()` macros:

```
error[E0433]: failed to resolve: use of undeclared type or module `El`
  --> src/parser.rs:23:9
   |
23 |         dbg!(s);
   |         ^^^^^^^^ use of undeclared type or module `El`
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
```

**Comment by David-OConnor** [comment]
Fixed in the latest release; 0.2.7
